subsys/bluetooth/audio/cap_initiator.c

```diff
@@ -944,8 +944,11 @@ void bt_cap_initiator_codec_configured(struct bt_cap_stream *cap_stream)
 			continue;
 		}
 
-		__ASSERT(free_conn, "No free conns");
-		*free_conn = stream_conn;
+		/* Have both assert and `if` to satisfy coverity */
+		__ASSERT(free_conn != NULL, "No free conns");
+		if (free_conn != NULL) {
+			*free_conn = stream_conn;
+		}
```

Comment:
LGTM, but maybe you would like this one:
```suggestion
		if (free_conn != NULL) {
			*free_conn = stream_conn;
		} else {
			__ASSERT_PRINT("No free conns");
		}
```